### PR TITLE
Use botkube install, fix sync:cli, remove certs flag from MM

### PR DESCRIPTION
**Pull request by mszostok**
## Description

Changes proposed in this pull request:

- Use botkube install, 
- Update install instructions,
- Fix `sync:cli`, 
- Remove certs flag from MM

Preview URL: https://switch-to-cli.botkube-docs.pages.dev/next/

### CODE REVIEW

File: docs/cli/getting-started.mdx [hunk 1]
```diff
@@ -58,11 +46,19 @@ For Windows, you will add it to the `%PATH%`.
...
+   export ARCH="" # allowed values: amd64, arm64, armv7
+   #export SUFFIX=".exec" # uncomment if OS is `windows`
+   curl -Lo botkube https://github.com/kubeshop/botkube/releases/download/v1.1.1/botkube_${OS}_${ARCH}${SUFFIX}
+   ```
+
 2. Move it to a location in the PATH. For example:
 
    ```shell
-   mv botkube_linux_amd64 /usr/local/bin/botkube
+   mv botkube /usr/local/bin/botkube
```
Comment (josefkarasek) [inline docs/cli/getting-started.mdx:61]: Will the bin already have exec permissin?

File: docs/cli/getting-started.mdx [hunk 2]
```diff
@@ -19,27 +19,15 @@ import TabItem from "@theme/TabItem";
...
 
 ```shell
-curl -Lo botkube https://github.com/kubeshop/botkube/releases/download/v1.1.1/botkube_linux_amd64
-```
-
-Copy to binary directory:
-
-```shell
-chmod +x botkube && mv botkube /usr/local/bin/botkube
+brew install kubeshop/botkube/botkube
```
Comment (josefkarasek) [inline docs/cli/getting-started.mdx:30]: I suspect not many linux users use brew on linux, I'd put bin download here.

File: docs/cli/getting-started.mdx (hunk 1 above)
Comment (mszostok) [reply docs/cli/getting-started.mdx:61]: good point, I restored `chmod +x botkube &&`

File: docs/cli/getting-started.mdx (hunk 2 above)
Comment (mszostok) [reply docs/cli/getting-started.mdx:30]: fixed 👌

**Comment by mszostok** [comment]
Thanks @josefkarasek  for the review! 

Here is a commit with fixes https://github.com/kubeshop/botkube-docs/pull/241/commits/72842e0783e89eda83facde46be9e6ef2b0988ff

**Review by josefkarasek — APPROVED** [review]
lgtm

**Pull request merged**